### PR TITLE
add process to fix contract with code

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/blockscout/issues/1389

### CODE REVIEW

**Comment by goodsoft** [comment]
The results of the two queries should be combined and deduped, shouldn't they?
Otherwise, we _might_ schedule of the same block twice.

Also, just to make it clear, which fetchers do normally fetch the data in question?
I think it would be a good idea to double-check, whether all of them behave correctly now.

**Comment by ayrat555** [comment]
@goodsoft there only 12 addresses for the first query. and more than `6_000_000` for the second query.
```
SELECT COUNT(DISTINCT ad.hash) FROM addresses ad
LEFT JOIN transactions tr ON ad.hash = tr.created_contract_address_hash
WHERE ad.hash IS NOT NULL AND ad.contract_code IS NULL AND tr.error IS NULL;
```

The issue is caused by the fact that we didn't set contract code if it was created in the main transaction

it's fixed in https://github.com/poanetwork/blockscout/pull/1577/files#diff-72a28f68611450b759a1bb127dca6afeR90

File: apps/indexer/lib/indexer/temporary/addresses_without_code.ex [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  require Logger
+
+  import Ecto.Query
+
+  alias Explorer.Chain.{Address, Block, Transaction}
+  alias Explorer.Repo
+  alias Indexer.Block.Realtime.Fetcher
+  alias Indexer.Temporary.AddressesWithoutCode.TaskSupervisor
+
+  @task_options [max_concurrency: 3, timeout: :infinity]
```
Comment (vbaranov) [inline apps/indexer/lib/indexer/temporary/addresses_without_code.ex]: @ayrat555 should we define a finite timeout and batch size for both select queries?

Comment (ayrat555) [reply apps/indexer/lib/indexer/temporary/addresses_without_code.ex]: https://github.com/poanetwork/blockscout/pull/1577/files#diff-474a4b9bd73dab720bead8a6acadb7f7R62 by default it selects by 500 records

**Pull request merged**